src/coreclr/inc/corjit.h

```diff
@@ -117,6 +117,8 @@ enum CorJitAllocMemFlag
     CORJIT_ALLOCMEM_FLG_RODATA_16BYTE_ALIGN = 0x00000002, // The read-only data will be 16-byte aligned
     CORJIT_ALLOCMEM_FLG_32BYTE_ALIGN   = 0x00000004, // The code will be 32-byte aligned
     CORJIT_ALLOCMEM_FLG_RODATA_32BYTE_ALIGN = 0x00000008, // The read-only data will be 32-byte aligned
+    CORJIT_ALLOCMEM_FLG_64BYTE_ALIGN   = 0x00000010, // The code will be 64-byte aligned
+    CORJIT_ALLOCMEM_FLG_RODATA_64BYTE_ALIGN = 0x00000020, // The read-only data will be 64-byte aligned
```

Comment:
Yes. Without it you can wind up with every access being a cache line split.

Reply:
So, that applies to data. What about code?

Also, for data, do we already 32-byte align data sections that have SIMD32 constants?

Reply:
> What about code?

Both the Intel and AMD optimization manuals have suggestions detailing when to use 64-byte alignment for code (such as method bodies or loops)

However, such support is not tied to AVX-512 and so I don’t think it’s necessary to include as part of this PR and is instead something we may want to look at independently.

> Also, for data, do we already 32-byte align data sections that have SIMD32 constants?

Yes. Such support was added some time back.

Reply:
@tannergooding To clarify - is the suggestion here to keep `CORJIT_ALLOCMEM_FLG_RODATA_64BYTE_ALIGN` and remove `CORJIT_ALLOCMEM_FLG_64BYTE_ALIGN`

FWIW, my understanding of what happens now : I added `CORJIT_ALLOCMEM_FLG_64BYTE_ALIGN` but it's not actively used since I'm not setting the flag(refer : https://github.com/dotnet/runtime/blob/main/src/coreclr/jit/emit.cpp#L6406-L6415 )

Reply:
Right. I think we want to just leave it off rather than having some presently unused code.

It may be beneficial to support such alignment in the future, but since its a general purpose thing unrelated to AVX-512, I think it should be done independently.